metron-interface/metron-rest/src/main/scripts/metron-rest.sh

```diff
@@ -83,7 +83,7 @@ echo "METRON_SPRING_PROFILES_ACTIVE=${METRON_SPRING_PROFILES_ACTIVE}"
 
 # the vagrant Spring profile provides configuration values, otherwise configuration is provided by rest_application.yml
 if [[ !(${METRON_SPRING_PROFILES_ACTIVE} == *"vagrant"*) ]]; then
-    METRON_CONFIG_LOCATION=" --spring.config.location=$METRON_HOME/config/rest_application.yml,classpath:/application.yml"
```

Comment:
The ordering/appending via `--spring.config.additional-location` makes sense. But I see you've removed `classpath:/application.yml` altogether. Was that intentional? How is application.yml loaded now, or is it not needed at all?

Reply:
Using `--spring.config.location` replaces the default locations.  This is why `classpath:/application.yml` was originally included even though it matches the default locations.  Using `--spring.config.additional-location` adds to the list of configs instead of replacing the default locations so having `classpath:/application.yml` in the list is no longer necessary.  Doing it this way enforces that anything added on top with `--spring.config.additional-location` will take priority over the default.  I think this is the correct way to do it because you don't have to worry about ordering them correctly.

Reply:
@merrimanr - perfect, that makes sense. Thanks for the explanation! +1 by inspection.